### PR TITLE
Rds terraform

### PR DESCRIPTION
**Pull request by S-Khan2**
Hey Ahmed,

I've just added some terraform to provision a RDS instance. Please have a look and merge to main if it looks good.

Sahl

### CODE REVIEW

**Comment by ahmedm89** [comment]
hey Sahl, 
yep all checkout mate, will merge to main now

**Review by ahmedm89 — COMMENTED** [review]
looks good

**Review by ahmedm89 — COMMENTED** [review]
all good

File: .gitignore [hunk 1]
```diff
@@ -1,13 +1,11 @@
 .venv/*
 .env
 *.pem
+*.zip
```
Comment (ahmedm89) [inline .gitignore:4]: this looks good

**Review by ahmedm89 — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (S-Khan2) [reply .gitignore:4]: Great!

**Comment by ahmedm89** [comment]
looks good sahl , will merge now

**Pull request merged**